extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1472,11 +1476,15 @@ export class AmpA4A extends AMP.BaseElement {
         metaData.customStylesheets.forEach(stylesheet => {
           if (!isObject(stylesheet) || !stylesheet['href'] ||
               typeof stylesheet['href'] !== 'string' ||
-              !/^https:\/\//i.test(stylesheet['href'])) {
+              !isSecureUrl(stylesheet['href'])) {
             throw new Error(errorMsg);
           }
         });
       }
+      if (isArray(metaDataObj['images'])) {
+        // Load maximum of 5 images.
+        metaData.images = metaDataObj['images'].splice(0, 5);
```

Comment:
This does not match the comment! `#slice` is not `#splice`.

Reply:
Actually, this will work since `#splice` returns the removed elements. But still, not the canonical use, `#slice` would be better here.